### ISSUE TITLE
[nuclide-swift] "An error occurred when invoking SourceKitten" (autocompletion)

### ISSUE THREAD

**Issue by pkrll** [facebook/nuclide#1552]
### Issue and Steps to Reproduce

If I have Sourcekitten enabled, and try to write something Atom throws an error, stating that "an error occurred when invoking SourceKitten".

I installed Nuclide through Atom, SourceKitten with Homebrew (symlinked in /usr/local/bin). I did build the project using Nuclide first.

**Nuclide-swift settings:**

<img width="744" alt="screen shot 2018-06-14 at 00 47 00" src="https://user-images.githubusercontent.com/140205/41382452-80911108-6f6c-11e8-88f7-0601da690adf.png">

### Expected Behavior

I should get autocompletion.

### Actual Behavior

I get an error:

<img width="445" alt="screen shot 2018-06-14 at 00 41 01" src="https://user-images.githubusercontent.com/140205/41382254-abcb8b74-6f6b-11e8-9228-43b76a0e501c.png">

```
An error occurred when invoking SourceKitten
Please file a bug.
exit code: 1
stdout:


stderr:
The operation couldn’t be completed. (SourceKittenFramework.Request.Error error 2.)

command:
complete --text import Foundation

let fileManager = FileManager.default
--offset 19 -- -swift-version 4 -target x86_64-apple-macosx10.10 -sdk /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.13.sdk -F /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/Library/Frameworks -Onone -g -enable-testing -j4 -DSWIFT_PACKAGE -module-cache-path /Users/path/to/project/.build/x86_64-apple-macosx10.10/debug/ModuleCache (+ a bunch of other files in the project...)

```

### Versions
* Atom: 1.27.2
* Nuclide: 0.315.0
* Client OS: macOS High Sierra 10.13.5

### Additional Details
* Installed packages (`apm ls --installed`):
├── advanced-open-file@0.16.8
├── atom-ide-ui@0.12.0 (disabled)
├── atom-material-syntax@1.0.8
├── atom-material-ui@2.1.3
├── atomic-emacs@0.12.1
├── autocomplete-python@1.10.5 (disabled)
├── build@0.70.0 (disabled)
├── build-make@0.13.0 (disabled)
├── busy-signal@1.4.3
├── dash@1.7.1
├── deep-space-syntax@1.1.0
├── doc-green-syntax@1.1.0
├── doc-green-ui@1.1.0
├── docblockr@0.13.7
├── docs-snippets@0.8.0 (disabled)
├── editorconfig@2.2.2
├── file-icons@2.1.21
├── git-time-machine@2.1.0 (disabled)
├── ide-java@0.8.3 (disabled)
├── ide-php@0.7.14 (disabled)
├── intentions@1.1.5 (disabled)
├── language-apache@1.8.0 (disabled)
├── language-applescript@0.3.0 (disabled)
├── language-haskell@1.17.7 (disabled)
├── language-htaccess@0.2.0 (disabled)
├── language-latex@1.1.1 (disabled)
├── language-mips@0.3.0 (disabled)
├── language-restructuredtext@1.1.2 (disabled)
├── language-rust@0.4.12 (disabled)
├── language-swift@0.5.0
├── language-vue@0.23.1 (disabled)
├── language-vue-component@0.5.0 (disabled)
├── latex@0.50.0 (disabled)
├── line-count@0.5.0
├── linter@2.2.0 (disabled)
├── linter-clang@4.1.2 (disabled)
├── linter-javac@1.9.4 (disabled)
├── linter-rust@0.9.0 (disabled)
├── linter-swiftlint@1.3.1
├── linter-ui-default@1.7.1 (disabled)
├── material-syntax@0.6.1
├── minimap@4.29.8
├── monokai@0.24.0
├── multi-cursor@2.1.5
├── nucleus-dark-ui@0.12.3
├── nuclide@0.315.0
├── packages
├── remote-atom@1.3.12 (disabled)
├── remote-ftp@2.2.0 (disabled)
├── rusty-dark-syntax@0.7.0
├── script@3.18.1
├── set-syntax@0.4.0
├── seti-syntax@1.1.3
├── seti-ui@1.10.0
├── slime@2.4.0
├── swift-debugger@0.1.2 (disabled)
├── teletype@0.13.3 (disabled)
├── todo-show@2.3.0
└── wakatime@7.0.4

**Comment by joscdk** [comment]
I am experiencing same error, i have tried both with:

```
brew install sourcekitten
```
(Installs v. 0.20.0)

And by manually building sourcekitten with
```
swift build
```
(Installs v. 0.21.0)

**Comment by yeswolf** [comment]
The patch attached fixes this issue and probably #1002
[completion.txt](https://github.com/facebook/nuclide/files/2349276/completion.txt). Can't dedicate my time to the pull request now, probably someone can integrate it.
